cmd/event-received/main.go

```diff
@@ -35,6 +35,10 @@ type moreEvidenceRequiredEvent struct {
 	UID string `json:"uid"`
 }
 
+type feeDeniedEvent struct {
+	UID string `json:"uid"`
+}
```

Comment:
maybe we should have had one `uidEvent`

Reply:
It is becoming quite apparent now...I was planning on holding off until the event catalogue is fully populated to see if there are any unexpected useful data points they are planning on sending over but seeing as everything working on just a UID makes sense to consolidate now 👍